client/web/src/enterprise/campaigns/settings/backend.ts

```diff
@@ -61,13 +63,14 @@ export function deleteCampaignsCredential(id: Scalars['ID']): Promise<void> {
 }
 
 export const queryUserCampaignsCodeHosts = ({
+    username,
     first,
     after,
 }: Partial<UserCampaignsCodeHostsVariables>): Observable<CampaignsCodeHostsFields> =>
```

Comment:
```suggestion
}: UserCampaignsCodeHostsVariables): Observable<CampaignsCodeHostsFields> =>
```
This can go now that the query function to the filtered connection is wrapped in a useCallback that does this. That also means that you can save the `username ?? null` up there